docs/ref/validators.txt

```diff
@@ -161,9 +161,11 @@ to, or in lieu of custom ``field.clean()`` methods.
 
     .. attribute:: schemes
 
-        URL/URI scheme list to validate against. If not provided, the default
-        list is ``['http', 'https', 'ftp', 'ftps']``. As a reference, the IANA
-        website provides a full list of `valid URI schemes`_.
+        URL/URI scheme list to validate against. If not provided, the
+        default is ``['http', 'https', 'ftp', 'ftps']``. If the special
+        valid ``__all__`` is passed, then all schemas are allowed.
```

Comment:
Need a `.. versionchanged:: 1.10` annotation and a mention in the release notes. See the [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist).